frontend-react/src/components/MessageTracker/MessageDetails.test.tsx

```diff
@@ -191,7 +191,7 @@ describe("RSMessageDetail component", () => {
         ).toBeVisible();
         expect(screen.getAllByText("Date/Time Submitted")[0]).toBeVisible();
         expect(screen.getByText("09/28/2022, 10:21:33 PM")).toBeVisible();
-        expect(screen.getByText("File Location")).toBeVisible();
+        expect(screen.getAllByText("File Location")[0]).toBeVisible();
```

Comment:
There are multiple elements on the page with `File Location` in the name now so we must get the array of them and index.